R/gibbs-sampler.R

```diff
@@ -1,65 +0,0 @@
-################################################################################
-
-#' Gibbs Sampler for the truncated multivariate normal distribution
-#'
-#' @param n_sim Number of draws from the Gibbs sampler (after burn-in).
-#' @param sigma Covariance matrix for the multivariate normal distribution.
-#' @param lower Lower truncation for the normal distribution. It can be `-Inf`.
-#' @param upper Upper truncation for the normal distribution. It can be `Inf`.
-#' @param fixed Should variables be fixed? Logical vector with an entry for each 
-#'   dimension. Default is `TRUE` when `upper` is equal to `lower`.
-#' @param ind Indices of variables to return from the Gibbs sampler. 
-#'   Default is `1` and corresponds to the first variable (usually the genetic 
-#'   liability of the first phenotype). Use `c(1, 5)` to return both the first and 
-#'   fifth variables.
-#' @param burn_in Number of iterations that count as burn in for the Gibbs sampler.
-#'
-#' @return The sampling values from the Gibbs sampler for the variables `ind`.
-#' 
-#' @references
-#' Kotecha, J. H., & Djuric, P. M. (1999, March). Gibbs sampling approach for 
-#' generation of truncated multivariate gaussian random variables. In 1999 IEEE 
-#' International Conference on Acoustics, Speech, and Signal Processing. 
-#' Proceedings. ICASSP99 (Cat. No. 99CH36258) (Vol. 3, pp. 1757-1760). IEEE.
-#' \doi{10.1109/ICASSP.1999.756335}
-#' 
-#' Wilhelm, S., & Manjunath, B. G. (2010). tmvtnorm: A package for the truncated 
-#' multivariate normal distribution. The R Journal. \doi{10.32614/RJ-2010-005}
-#'
-#' @export
-#' 
-#' @examples
-#' samp <- rtmvnorm.gibbs(10e3, sigma = matrix(c(1, 0.2, 0.2, 0.5), 2),
-#'                        lower = c(-Inf, 0), upper = c(0, Inf), ind = 1:2)
-#' 
-rtmvnorm.gibbs <- function(n_sim, sigma, lower, upper, 
```

Comment:
Previous functions have been completely removed?

Reply:
They have not been removed, but renamed or merged with other functions. The function rtmvnorm.gibbs can be found in the R script The_Gibbs_Sampler.

Reply:
You must not remove or rename functions that are exported, as people might be already using them, and this would break existing code.